Makefile

```diff
@@ -20,6 +20,10 @@ GO_BUILD_PACKAGES :=$(strip \
 	./cmd/... \
 	$(wildcard ./tools/*) \
 )
+GO_TEST_PACKAGES :=$(strip \
+	./... \
+	./vendor/k8s.io/kubernetes/pkg/kubectl/... \
```

Comment:
let's do it properly since the beginning then
```
$(shell $(GO) list -f '{{range .Deps}}{{printf "%s\n" .}}{{end}}' ./cmd/... | grep --color=never -E '^github.com/openshift/oc/vendor/(sigs\.)?k8s\.io/')
```

Reply:
but i'd verify the timing